client/hotrod-client-legacy/src/test/java/org/infinispan/client/hotrod/test/MultiHotRodServersTest.java

```diff
@@ -72,6 +72,7 @@ protected org.infinispan.client.hotrod.configuration.ConfigurationBuilder create
             .host(host)
             .port(serverPort)
             .maxRetries(maxRetries())
+            .socketTimeout(Integer.MAX_VALUE)
```

Comment:
any problem with the server? If the legacy client is going to be removed in ISPN16, it is probably worth to open a PR and remove it.

Reply:
I think this is just a left over from when I was debugging a test. I'll revert